src/charm.py

```diff
@@ -7,7 +7,7 @@
 import logging
 import time
 
-from charms.rolling_ops.v0.rollingops import RollingOpsManager
+from charms.rolling_ops.v0.rollingops import ActiveStatus, RollingOpsManager
```

Comment:
This should not be imported from rollingops

Reply:
Indeed, it was a silent automatism, fixing.

Reply:
Fixed.